examples/convert_eeg_to_bids.py

```diff
@@ -155,21 +164,18 @@
 for subj_idx in [1, 2]:
     for task_idx in [2, 4, 12]:
         # Load the data
-        edf_path = os.path.join(data_dir,
-                                'S{:03}'.format(subj_idx),
-                                'S{:03}R{:02}.edf'.format(subj_idx, task_idx))
-        raw = read_raw_edf(edf_path, preload=True)
-
-        # `kind` and `trial_type` were already defined above
-        raw_to_bids(subject_id='{:03}'.format(subj_idx),
-                    task=task_names[task_idx],
-                    run=run_mapping[task_idx],
-                    raw_file=raw,
-                    output_path=output_path,
-                    kind=kind,
-                    event_id=trial_type,
-                    overwrite=True
-                    )
+        edf_path = eegbci.load_data(subject=subj_idx, runs=task_idx)[0]
```

Comment:
@sappelhoff I changed this a bit. I find this alternative more readable. Hope this is okay for you.

Reply:
+1 generally, however specifically I find it more readable to have each argument to a function on its own line. But that's just my opinion about this thing ... the remaining changes are fine.